src/utils/filters.js

```diff
@@ -139,25 +142,58 @@ export function generateFilterOptions(layerData) {
 
     // if datum passes all conditions
     if (doPushDatum) {
+      // loop through the fiters to add to counts of passing data
       for (let f = 0; f < layerFilter.length; f += 1) {
         filter = layerFilter[f];
         acceptedFilterValues = ((layerData.layerObj && layerData.layerObj.aggregate)
           || layerData.aggregate)['accepted-filter-values']
           && ((layerData.layerObj && layerData.layerObj.aggregate) || layerData.aggregate)['accepted-filter-values'][f];
 
-        if (!filterOptions[filter].filterValues[datum[filter]]) {
+        filterIsMultiSelect = acceptedFilterValues === 'multi' ||
+          (layerData['data-parse'][filter] && layerData['data-parse'][filter].type === 'multiple');
```

Comment:
i think we should make this check more specific by first checking if the `data-parse` prop exists in the mapSpec i.e
`(layerData['data-parse'] && layerData['data-parse'][filter] && layerData['data-parse'][filter].type === 'multiple')`